packages/react-image/Spec.md

```diff
@@ -0,0 +1,153 @@
+| Expands image frame to fill its parent container                        | `maximizeFrame`         | `fluid`         | partially matching            |
+| Optional callback for when the image load state changed                 | `onLoadingStateChanged` | N/A             | v0 missing                    |
+| Fades the image in when loaded                                          | `shouldFadeIn`          | N/A             | v0 missing                    |
+| The image starts as visible and is hidden on error                      | `shouldStartVisible`    | N/A             | v0 missing                    |
+| Provides customized styling that will layer on top of the variant rules | `styles`                | `styles`        | matching                      |
+| Theme provided by HOC                                                   | `theme`                 | `variables`     | matching                      |
+| Accessibility behavior overridden by the user                           | N/A                     | `accessibility` | v7/v8 missing                 |
+| Alternative text                                                        | N/A                     | `alt`           | v7/v8 missing                 |
+| ARIA label                                                              | N/A                     | `aria-label`    | v7/v8 missing                 |
+| An element type to render as (string or component)                      | N/A                     | `as`            | v7/v8 missing                 |
+| An image may be formatted to appear inline with text as an avatar       | N/A                     | `avatar`        | v7/v8 missing                 |
+| An image can appear circular                                            | N/A                     | `circular`      | v7/v8 missing                 |
+| Component design prop                                                   | N/A                     | `design`        | v7/v8 missing                 |
+| Image source URL                                                        | N/A                     | `src`           | v7/v8 missing                 |
+
+### v8 usage
+
+```jsx
+import { Image, ImageFit } from '@fluentui/react/lib/Image';
+
+<Image maximizeFrame imageFit={ImageFit.contain} src="..." alt="Example of the image component in v8" />;
+```
+
+### v0 usage
+
+```jsx
+<Image fluid src="..." alt="Example of the image component in v0" />
+```
+
+## Variants
+
+n/a
+
+## API proposal
+
+See [react-image/Image.types.ts] for the API.
+
+[react-image/image.types.ts]: https://github.com/microsoft/fluentui/blob/master/packages/react-image/src/components/Image/Image.types.ts
+
+| Name       | Type      | Default value | Comments                                     |
+| ---------- | --------- | ------------- | -------------------------------------------- |
+| `alt`      | `string`  | `undefined`   |                                              |
```

Comment:
`alt` would be in the html attribute typings right ?

Reply:
Right, removed this from spec.